src/utils.js

```diff
@@ -27,7 +30,57 @@ const getBrowser = (userAgent) => {
     return browser;
 };
 
+const getBrowsersWithVersions = (browserList) => {
+    const browsersWithVersions = {};
+
+    for (const browserDefinition of browserList) {
+        const [browser, version] = browserDefinition.split(' ');
+        if (!SUPPORTED_BROWSERS.includes(browser)) {
+            // eslint-disable-next-line no-continue
+            continue;
```

Comment:
Why not just do 
`if (SUPPORTED_BROWSERS.includes(browser)) { ... the rest of the loop ...  }`
instead of disabling linting so that you can use continue? :)

Reply:
I felt like it is gonna be too much code nesting 😄